src/main/java/org/camunda/community/benchmarks/utils/BpmnJobTypeParser.java

```diff
@@ -74,9 +72,9 @@ private static Set<String> extractJobTypesFromResource(Resource resource, String
                 for (ZeebeTaskDefinition taskDef : taskDefinitions) {
                     String type = taskDef.getType();
                     if (type != null && !type.trim().isEmpty()) {
-                        String resolvedType = resolveJobType(type, starterId);
-                        if (resolvedType != null) {
-                            jobTypes.add(resolvedType);
+                        String extractedType = extractStaticJobType(type.trim());
+                        if (extractedType != null) {
+                            jobTypes.add(extractedType);
```

Comment:
Still too complicated. Please add `!type.startsWith("=")` to the condition of the first if-statement and then directly call `jobTypes.add(type.trim())`. Delete the entire method `extractStaticJobType`.